src/Phinx/Db/Adapter/PostgresAdapter.php

```diff
@@ -92,6 +102,8 @@ public function connect(): void
                 );
             }
 
+            $this->useIdentity = (float)$db->getAttribute(PDO::ATTR_SERVER_VERSION) >= 10;
```

Comment:
I'm not sure if it's the right approach to decide whether to use an `identity` depending on the version of PostgreSQL. This makes it impossible to use the `serial` type to create auto-increment column. These are different things that work in different ways. And one of them does not replace the other. There must be a choice of which auto-increment approach will be used.